File: Lista de exercicio/projetoPYTHON/questao1/questao1.py
class Carro:
    def __init__(self, marca, modelo, ano):
        self.marca = marca
        self.modelo = modelo
        self.ano = ano

    def exibir_detalhes(self):
        return f"{self.marca} {self.modelo} ({self.ano})"

def executar_questao1():
    carro1 = Carro("Toyota", "hilux", 2024)
    carro2 = Carro("Kya", "sorento", 2015)
    carro3 = Carro("Mitsubishi ", "lancer", 2012)

    carros = [carro1, carro2, carro3]

    for carro in carros:
        print(carro.exibir_detalhes())
    

File: Lista de exercicio/projetoJAVA/src/projetojava/questao1.java
package projetojava;

 class Carro {
    String marca;
    String modelo;
    int ano;

    Carro(String marca, String modelo, int ano) {
        this.marca = marca;
        this.modelo = modelo;
        this.ano = ano;
    }

    String exibirDetalhes() {
        return marca + " " + modelo + " (" + ano + ")";
    }
}

public class questao1 {
    public static void main(String[] args) {
        Carro carro1 = new Carro("Toyota", "hilux", 2024);
        Carro carro2 = new Carro("Kya", "sorento", 2015);
        Carro carro3 = new Carro("Mitsubishi", "lancer", 2012);

        Carro[] carros = {carro1, carro2, carro3};

        for (Carro carro : carros) {
            System.out.println(carro.exibirDetalhes());
        }
    }
}
